nilimamaka/SpringBoot/MultipleDBConnection/src/main/java/com/example/multipledbconnection/fileDetails/exception/IdNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.multipledbconnection.fileDetails.exception;
+
+public class IdNotFoundException extends RuntimeException{
```

Comment:
I think this exception class is for throwing exception when a File is not found by Id. So better to throw FileNotFoundException rather IdNotFound.